client/cypress/end-to-end/ballot-comparison-failure-cases.spec.js

```diff
@@ -0,0 +1,238 @@
+import 'cypress-file-upload'
+
+before(() => cy.exec('./cypress/seed-test-db.sh'))
+
+describe('Ballot Comparison Failure Cases', () => {
+  const auditAdmin = 'audit-admin-cypress@example.com'
+  const jurisdictionAdmin = 'wtarkin@empire.gov'
+  const uuid = () => Cypress._.random(0, 1e6)
+  let id = 0
+  let board_credentials_url = ''
+
+  it('CSV and input errors', () => {
+    id = uuid()
+    cy.visit('/')
+    cy.loginAuditAdmin(auditAdmin)
+    cy.get('input[name=auditName]').type(`TestAudit${id}`)
+    cy.get('input[value="BALLOT_COMPARISON"]').check({ force: true })
+    cy.findByText("Create Audit").click()
+    cy.contains("Audit Setup")
+    cy.viewport(1000,1000)
+    // attempt uploading a jurisdiction filesheet without selecting one
```

Comment:
i think this case (and similar cases below for other files) makes more sense as a unit test, since it doesnt involve the backend

Reply:
Fair point. It might already cover it actually...